scss/_list-group.scss

```diff
@@ -37,6 +40,9 @@
   counter-reset: section;
 
   > .list-group-item::before {
+    // Boosted mod
+    margin-right: calc(var(--#{$prefix}list-group-item-padding-x) * 2 / 3); // stylelint-disable-line function-disallowed-list
```

Comment:
Just to stay consistent:

`margin-right: calc(var(--#{$prefix}list-group-item-padding-x) * 2 / 3); /* stylelint-disable-line function-disallowed-list */ // Boosted mod`